roles/cnf_cert/templates/tnf_config.yml.j2

```diff
@@ -65,23 +38,6 @@ targetCrdFilters:
 {% endif %}
 {% endfor %}
 
-{% if test_network_function_version is version('v4.4.0', '<=') %}
```

Comment:
This condition checks for tnf <= v4.4.0, and if we remove it then I guess is something we really do not need right?  just curious since this PR is just keeping compatibility for  >= v4.4.0